SpringProject/src/main/java/com/gradproject2019/auth/service/AuthServiceImpl.java

```diff
@@ -45,4 +46,16 @@ private Token createToken(Long userId) {
                 .withToken(UUID.randomUUID())
                 .build();
     }
+
+    @Override
+    public void logout(UUID token) {
+        checkTokenExists(token);
+        authRepository.deleteById(token);
```

Comment:
Is this method named correctly? It seems you're trying to delete by ID, but passing a token through instead